lib/widgets/rss_feed_page.dart

```diff
@@ -1,67 +1,95 @@
 import 'package:flutter/material.dart';
-import 'package:meta/meta.dart';
 import 'package:influx/config.dart';
 import 'package:influx/utility/rss_feed/rss_feed_reader.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:intl/intl.dart';
 
 /// A widget that displays contents from an RSS feed.
 class RssFeedPage extends StatefulWidget {
-
-  // TODO: this should not be injected manually but via a DI framework:
   final RssFeedReader rssFeedReader;
-
-  const RssFeedPage({this.rssFeedReader, Key key})
-      : super(key: key);
+  const RssFeedPage({this.rssFeedReader, Key key}) : super(key: key);
 
   @override
   _RssFeedState createState() => _RssFeedState(rssFeedReader: rssFeedReader);
 }
 
 class _RssFeedState extends State<RssFeedPage> {
-  RssFeedReader _rssFeedReader;
+  final int _maxRssFeedItems = 20;
+  final _formatter = DateFormat('dd.MM.yyyy HH:mm:ss');
+
+  final RssFeedReader _rssFeedReader;
+  final ScrollController _scrollController = ScrollController();
 
-  _RssFeedState({rssFeedReader})
+  int _lastRssFeedIndex = 0;
+
+  _RssFeedState({RssFeedReader rssFeedReader})
       : _rssFeedReader =
             rssFeedReader ?? RssFeedReader(url: InFluxConfig.rssFeedUrl);
 
   @override
   Widget build(BuildContext context) {
-    final maxRssFeedItems = 20;
-    var rssPosts = _rssFeedReader.fetchRssPosts(maxRssFeedItems);
```

Comment:
@patrickp89 If the users scrolls down the list, all posts which were already fetched from the rss-feed-source will be re-fetched. I would suggest to only fetch the new items and to successively add them to the list of posts, instead of fetching the whole feed in the `rssposts`-Future again and again.